samples/guide/build.gradle.kts

```diff
@@ -8,10 +8,15 @@ dependencies {
   implementation(projects.mockwebserver)
   implementation(projects.okhttpTestingSupport)
   implementation(projects.okhttpTls)
+  implementation(projects.loggingInterceptor)
   implementation(libs.animalsniffer.annotations)
   implementation(libs.squareup.moshi)
   implementation(libs.squareup.okio.fakefilesystem)
   kapt(libs.squareup.moshi.compiler)
+
+  implementation("org.testcontainers:testcontainers:1.17.6")
+  implementation("com.github.mike10004:fengyouchao-sockslib:1.0.6")
```

Comment:
What’s with the group name on this? Shouldn’t it be https://github.com/fengyouchao/sockslib ?

(I’m anxious about the implicit endorsement of this library)

Reply:
It’s kind of kitchen-sinky

https://github.com/fengyouchao/sockslib/blob/master/src/main/java/sockslib/utils/mongo/MongoDBUtil.java

Reply:
It's not officially published.